Sources/BraveStrings/BraveStrings.swift

```diff
@@ -4340,6 +4340,12 @@ extension Strings {
     value: "No Saved Website Data",
     comment: "Shown when the user has no website data (such as cookies, caches, or local storage) persisted to their device."
   )
+  
+  public static let blockCookieConsentNotices = NSLocalizedString(
+      "BlockCookieConsentNotices",
+      tableName: "BraveShared", bundle: .module,
+      value: "Block Cookie Consent Notices",
+      comment: "A title for a setting that enables cookie consent notices")
```

Comment:
Do we need this?

Reply:
yes its used in the shields view as well